install.js

```diff
@@ -161,3 +155,15 @@ function getIeDriverPlatform() {
     return new Error('Architecture not supported');
   }
 }
+
+exports.install = function(callback) {
+  async.series([
+    setup,
+    download,
+    chmodChromeDr.bind(null, conf.chromeDr.path)
+  ], callback);
+}
+
+if (require.main === module) {
```

Comment:
If you run it from the command line, it calls that function, if its required then nothing happens